apps/discord-bot/internal/command/addcmdaliascommand.go

```diff
@@ -0,0 +1,42 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/senchabot-opensource/monorepo/apps/discord-bot/internal/db"
+	"github.com/senchabot-opensource/monorepo/packages/gosenchabot"
+)
+
+func (c *commands) AddCmdAliasCommand(ctx context.Context, s *discordgo.Session, i *discordgo.InteractionCreate, db db.MySQL) {
+	options := i.ApplicationCommandData().Options
+
+	cmdName := options[0].StringValue()
+	params := strings.Split(options[1].StringValue(), " ")
+
+	cmdName = strings.ToLower(cmdName)
+	cmdName = gosenchabot.TrimExclamationPrefix(cmdName)
+	aliasCommands := gosenchabot.MakeUniqueArray(params)
+
```

Comment:
To avoid code duplication, an `options wrapper` can be used to check and get these values. I don't know if it has ever been duplicated, but for more maintainable code, `cmdName` and `params` getter can be methods with validations.